Pear/Controller/MatchViewController.swift

```diff
@@ -143,17 +142,17 @@ class MatchViewController: UIViewController {
         }
 
         NetworkManager.shared.getUser(netId: pairNetId).observe { response in
-            switch response {
-            case .value(let result):
-                guard result.success else {
-                    print("Network error: could not get user's pair.")
-                    return
-                }
-                DispatchQueue.main.async {
+            DispatchQueue.main.async {
+                switch response {
+                case .value(let result):
+                    guard result.success else {
+                        print("Network error: could not get user's pair.")
+                        return
+                    }
                     closure(result.data)
```

Comment:
I would call it something more descriptive (and consistent with other places), like `completion`